simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListRegistryViewModel.kt

```diff
@@ -59,11 +59,21 @@ class AccountListRegistryViewModel(private val locationManager: LocationManager)
       this.profilesController.accountEvents()
         .observeOn(AndroidSchedulers.mainThread())
         .subscribe(this::onAccountEvent),
-      Observable.combineLatest(queries, locationUpdates) { query, _ -> createQuery(query) }
-        .switchMapCompletable(this::executeQuery)
-        .subscribeOn(Schedulers.io())
+      /**
+       * Here we are hiding Library Finder functionality
+       */
+      queries.map {
```

Comment:
This seems kind of weird. Why has the above statement been unrolled like this? It doesn't seem to actually change behaviour...

Reply:
@Dumblydore @io7m check latest commit